terraform/cp4ba/README.md

```diff
@@ -1,124 +1,288 @@
-# Cloud Pak for Business Automation
+Everything is automated with Makefiles. However, instructions to get the
```

Comment:
This README contains outdated information.  See https://github.com/ibm-hcbt/cloud-pak-sandboxes/blob/joel_cp4ba_sandbox/terraform/cp4i/README.md for an example of the correct information that should be contained in this README.